understanding/21/reflow.html

```diff
@@ -12,68 +12,197 @@ <h1>Understanding Reflow</h1>
-      <figure id="figure-CSS-pixel-letter-4x"><img src="img/css-pixel-by-device.png" width="462" height="198" alt="Diagram showing the size of character needed by viewing distance to make the same image on the retina with small screen devices close, large screen devices further away."/>
-        <figcaption>A letter of the same CSS pixel size on different displays with different resolutions</figcaption>
-      </figure>
-        <p>When we read, the size of the print is not as important as the image it projects on the retina of our eye. Phones are designed for close viewing while desktops are designed for viewing farther away. As a consequence 16px print on a phone is physically smaller than 16px print on a desktop. This is not a problem because both print sizes cast the same image on our retina if they are viewed at their intended distance.</p>
-      </section>
-      <section>
-      <h3>Visibility and availability of content</h3> 
-      <p>How much of the content is visible may change at different scales. For example, navigation menus that are fully visible in the desktop layout are often collapsed into fewer items, or even into a single menu button (the 'hamburger' icon pattern) so they take up less screen space.</p>
-      <p>The Success Criterion is met as long as all content and functionality are still fully available -  either directly, or revealed via accessible controls, or accessible via direct links.</p>
-      </section>
+        <p>The intent of the Reflow success criterion is to allow users who need to enlarge (zoom in) content, to read multiline text content without having to scroll in multiple directions. Lines of text that extend beyond the sizing requirements defined by this success criterion will result in the need for such scrolling, the act of which can significantly increases both the physical and cognitive effort required to read. Decreasing the necessary effort can help mitigate the likelihood of a user losing their place when they reach the end of a non-reflowing line, and then need to scroll back to the start of the next line to continue reading.</p>
+
+        <p>The success criterion applies to both horizontally and vertically written languages. For pages where the primary written language expects vertical scrolling by default (such as English) it would not be expected, or generally desired, for the page or a section of content of the page to require horizontal scrolling to read lines of text when the content could have reasonbly reflowed to fit within a 320 CSS pixel wide area. Similarly, it would not be expected for the page, or a section of content within a page, to require vertical scrolling to read lines of text when the primary written language would expect horizontal scrolling for reading.</p>
```

Comment:
```suggestion
        <p>The success criterion applies to both horizontally and vertically written languages. For pages where the primary written language expects vertical scrolling by default (such as English) it would not be expected, or generally desired, for the page or a section of content of the page to require horizontal scrolling to read lines of text when the content could have reasonably reflowed to fit within a 320 CSS pixel wide area. Similarly, it would not be expected for the page, or a section of content within a page, to require vertical scrolling to read lines of text when the primary written language would expect horizontal scrolling for reading.</p>
```